hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -74,18 +112,11 @@ healthyOpenHeadTxOut =
   mkHeadOutput testNetworkId Fixture.testPolicyId headTxOutDatum
     & addParticipationTokens healthyParties
  where
-  headTxOutDatum = toUTxOContext (mkTxOutDatum healthyCloseDatum)
+  headTxOutDatum = toUTxOContext (mkTxOutDatum healthyOpenHeadDatum)
 
-healthySlotNo :: SlotNo
-healthySlotNo = arbitrary `generateWith` 42
-
-healthyClosingSnapshot :: ClosingSnapshot
-healthyClosingSnapshot =
-  CloseWithConfirmedSnapshot
-    { snapshotNumber = healthySnapshotNumber
-    , closeUtxoHash = UTxOHash $ hashUTxO @Tx healthyCloseUTxO
-    , signatures = healthySignature healthySnapshotNumber
-    }
+-- FIXME: This is not a healthy value anyhow related to the 'healthyCloseTx' above
+brokenSlotNo :: SlotNo
+brokenSlotNo = arbitrary `generateWith` 42
```

Comment:
this is not being used

Reply:
Thank you, that was the point of the previous commits to remove that... oops! forgot to actually remove it ;)